### PR TITLE
Replacing HighlightedText with SpecialText

### PR DESCRIPTION
**Pull request by ericlin11354**
[Here](https://user-images.githubusercontent.com/50276107/151652563-437364ce-3666-41f4-a14c-e90576ebeedb.mp4) is a demo. The general functionality of the chat and text bubbles should remain the same.
Currently working on using SpecialText onClick to toggle dropdown rather than using dropdownButton.

Note: Temporarily changed VoiceFlow API blocks URLs to test VUI with local database.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "eslint-config-next": "11.1.2",
     "jest": "^27.4.4",
     "react-test-renderer": "^17.0.2",
-    "typescript": "4.4.4"
+    "typescript": "4.4.4",
+    "webpack": "^5.67.0"
```
Comment (ralph-dev) [inline package.json:45]: Why was this added btw?

File: pages/order.tsx [hunk 2]
```diff
@@ -157,120 +165,101 @@ const Landing: NextPage = () => {
...
-      listProps: {
-        // dropdownButton: <div/>,
+  ): React.ReactElement => {
+    const icon = isFromBot ? Robot : User;
+
+    const getTextBubble = (): React.ReactElement | null => {
+
+      // no variables
+      if (!list)
+        return <StyledSmallText isFromBot={isFromBot} >{text}</StyledSmallText>
```
Comment (ralph-dev) [inline pages/order.tsx:198]: Put the surrounding brackets if your putting the word return

File: pages/order.tsx [hunk 3]
```diff
@@ -157,120 +165,101 @@ const Landing: NextPage = () => {
...
         </TextBubble>
       </TextBubbleContainer>
     );
 
   };
 
-  const displayHighlightedText = (): React.ReactElement => {
-    return <>{highlightedStrings}</>;
+  const getTextBubbles = (): React.ReactElement => {
+    return <>{textBubbles}</>;
```
Comment (ralph-dev) [inline pages/order.tsx:262]: Can't you just return without fragments?

File: pages/order.tsx [hunk 4]
```diff
@@ -642,6 +635,26 @@ const Landing: NextPage = () => {
...
+`;
+
+const StyledSmallText = styled(SmallText)<{ isFromBot: boolean}>`
+  ${({ isFromBot }): string => `
+    text-align: ${isFromBot ? 'left' : 'right'};
+    color: ${isFromBot ? 'black' : 'white'};
+  `}
+  white-space: pre-line;
+  margin: 0 2px 0 2px;
+`
```
Comment (ralph-dev) [inline pages/order.tsx:656]: If there is styled components in your application it means you can convert to components in storybook. I'll talk about this today for our lesson

**Review by ralph-dev — APPROVED** [review]
Lgtm

File: package.json (hunk 1 above)
Comment (ericlin11354) [reply package.json:45]: I was unsuccessful running the VUI with SpecialText inside react-ui's `master` branch. I was getting the `fetchcompilewasmtemplateplugin is not a constructor` error which I believe is related to webpack. To resolve this issue, I packaged SpecialText with react-ui's `vui-export`. I believe Jianjia (the creator of the branch) installed some plugins which resolved this webpack issue, hence why we add webpack here.

File: pages/order.tsx (hunk 3 above)
Comment (ericlin11354) [reply pages/order.tsx:262]: textBubbles is an array of React.ReactElements so it needs to be wrapped in a parent element. I will replace empty fragments with a div fragment.

File: package.json (hunk 1 above)
Comment (ralph-dev) [reply package.json:45]: Ah that error is related to workers.ts no worries

**Review by ralph-dev — APPROVED** [review]
Lgtm

**Comment by ralph-dev** [comment]
@ericlin11354  This repository has been deprecated please move the code to the CE MonoRepo
